### ISSUE TITLE
Error connecting

### ISSUE THREAD

**Issue by staurakis** [jabbink/PokemonGoBot#1248]
29 ┴Ϋή 20:27:02 [default] - Logging in to game server...
       29 ┴Ϋή 20:27:44 [default] - Server returned unexpected error: Failed to receive contents from server
29 ┴Ϋή 20:27:44 [default] - Retrying...
       Exception in thread "default" java.lang.IllegalStateException: Server refused your login credentials. Are they correct?
        at ink.abb.pogo.scraper.MainKt.startBot(Main.kt:202)
        at ink.abb.pogo.scraper.MainKt.startDefaultBot(Main.kt:154)
        at ink.abb.pogo.scraper.PokemonGoBotApplication$BotRunner$run$1.invoke(PokemonGoBotApplication.kt:80)
        at ink.abb.pogo.scraper.PokemonGoBotApplication$BotRunner$run$1.invoke(PokemonGoBotApplication.kt:67)
        at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:18)

**Comment by jabbink** [comment]
Hi @staurakis,

It looks like there is some information missing from your ticket that will be needed in order to diagnose and fix the problem at hand. Please take a look at the [Contribution Guidelines](https://github.com/jabbink/PokemonGoBot/blob/develop/.github/CONTRIBUTING.md) and [default Issue template (the one you did not follow)](https://github.com/jabbink/PokemonGoBot/blob/develop/.github/ISSUE_TEMPLATE.md), which will tell you **exactly** what your ticket has to contain in order to be processable.

I'm marking this one now as needing some more information. Please understand that if you do not provide that information within the next two days (until 2016-08-31 19:35) I'll close this ticket so it doesn't clutter the bug tracker.

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being.

**Comment by staurakis** [comment]
Sorry, i remotely opened this in order to copy the logs. The bot run without any issues for hours.
The i stopped it and run it once again after few hours. After some time that it run again smoothly, i received  a continous message that it was looting the same pokestop. I stopped it and tried to run again and i received the connection error above.
The account can login with the Niantic client without issues.
I did a clean buid and run it again but the outcome was the same. Any ideas?
